### DISCUSSION TITLE
版本不一样，向下兼容差距为何那么大？

### DISCUSSION THREAD

**Discussion by wch66** [HandyOrg/HandyControl#1323] [Q&A] [answered]
最近讲之前项目升级到最新版本的HandyControl，会有很多资源属性找不到，或者无法对应？这个有文档说明差异么？

**Comment by NaBian** [comment, marked as answer]
资源找不到要分清楚是 HC demo 中的资源还是 HC 自带的资源，属性找不到就是做了优化，HC 从创立到现在主要由我个人维护，个人的时间和能力有限，一些早期的设计思路会存在不合理之处，一定有很强烈的理由才促使我做不兼容的改动。此外，文档方面也欠缺很多，希望有时间和有意向的朋友们能帮助 HC 建立更好的文档体验。
